swarm/network/stream/snapshot_retrieval_test.go

```diff
@@ -67,6 +69,44 @@ func TestFileRetrieval(t *testing.T) {
 	}
 }
 
+// TestPureRetrieval tests pure retrieval without syncing
+// A configurable number of nodes and chunks
+// can be provided to the test.
+// A number of random chunks is generated, then stored directly in
+// each node's localstore according to their address.
+// Each chunk is supposed to end up at certain nodes
+// With retrieval we then make sure that every node can actually retrieve
+// the chunks.
+func TestPureRetrieval(t *testing.T) {
+	var nodeCount []int
+	var chunkCount []int
+
+	if *nodes != 0 && *chunks != 0 {
+		nodeCount = []int{*nodes}
+		chunkCount = []int{*chunks}
+	} else {
+		nodeCount = []int{16}
+		chunkCount = []int{16}
```

Comment:
Please increase to 100 or 150 or 200 chunks.

Anything above that gets slow, and we should understand why. At the same time we should have such slow tests on Travis, so 100-200 is somewhat a compromise. 16 however is too low in my opinion.